arch/x86_64/mcount-dynamic.c

```diff
@@ -255,6 +258,10 @@ static int patch_fentry_code(struct mcount_dynamic_info *mdi, struct uftrace_sym
 	unsigned char *insn = (void *)sym->addr + mdi->map->start;
 	unsigned int target_addr;
 
+	/* skip 'endbr64' instruction, which is inserted by (implicit) -fcf-protection option. */
+	if (!memcmp(insn, endbr64, sizeof(endbr64)))
+		insn += sizeof(endbr64);
```

Comment:
Yeah, maybe it's easier than changing it to iterate on the `__patchable_function_entries`.